src/App.tsx

```diff
@@ -1,14 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { FilterStatusType, Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+function filterFunction(
+  items: Todo[],
+  str: string,
```

Comment:
- `filterFunction`, really?) Is it the same as the filter method? The function name should start with a verb and `Function` in the name doesn't make any sense
- `items`/`str` are really bad names, `itemsInUse` is the same